docs/sources/mimir/reference-configuration-parameters/index.md

```diff
@@ -737,6 +737,11 @@ forwarding:
   # The CLI flags prefix for this block configuration is:
   # distributor.forwarding.grpc-client
   [grpc_client: <grpc_client>]
+
+# (experimental) Enable marking series as ephemeral based on the given matchers
+# in the runtime config.
+# CLI flag: -distributor.ephemeral-metrics-enabled
```

Comment:
Here it's called "ephemeral metrics" while below it's called "ephemeral series". Pick one of the two, so we standardise naming.

Reply:
thx, good catch. I started out calling it "ephemeral metrics", but then we changed the checker so it checks the entire labelset instead of only checking the metric name, so I'll rename it to "ephemeral series" everywhere.

Reply:
done here: 52465b710c2c347b675cf7c2fb959a8c9631f9ac